client/src/app/site/motions/components/motion-list/motion-list.component.html

```diff
@@ -92,7 +94,9 @@
 
                     <!-- recommendation -->
                     <span *ngIf="motion.recommendation && motion.state.next_states_id.length > 0">
-                        <mat-basic-chip class="bluegrey" [disabled]="true">{{ getRecommendationLabel(motion) }} </mat-basic-chip>
+                        <mat-basic-chip class="bluegrey" [disabled]="true"
+                            >{{ getRecommendationLabel(motion) }}
```

Comment:
This formatting is strange..

Reply:
that's my IDE doing it, I'm just the operator....